lib/switchtec.c

```diff
@@ -135,6 +135,36 @@ static const struct switchtec_device_id switchtec_device_id_tbl[] = {
 	{0x4252, SWITCHTEC_GEN4, SWITCHTEC_PAX},   //PAX 52XG4
 	{0x4236, SWITCHTEC_GEN4, SWITCHTEC_PAX},   //PAX 36XG4
 	{0x4228, SWITCHTEC_GEN4, SWITCHTEC_PAX},   //PAX 28XG4
+	{0x5000, SWITCHTEC_GEN5, SWITCHTEC_PFX},   //PFX 100XG5
+	{0x5084, SWITCHTEC_GEN5, SWITCHTEC_PFX},   //PFX 84XG5
+	{0x5068, SWITCHTEC_GEN5, SWITCHTEC_PFX},   //PFX 68XG5
+	{0x5052, SWITCHTEC_GEN5, SWITCHTEC_PFX},   //PFX 52XG5
+	{0x5036, SWITCHTEC_GEN5, SWITCHTEC_PFX},   //PFX 36XG5
+	{0x5028, SWITCHTEC_GEN5, SWITCHTEC_PFX},   //PFX 28XG5
+	{0x5100, SWITCHTEC_GEN5, SWITCHTEC_PSX},   //PSX 100XG5
+	{0x5184, SWITCHTEC_GEN5, SWITCHTEC_PSX},   //PSX 84XG5
+	{0x5168, SWITCHTEC_GEN5, SWITCHTEC_PSX},   //PSX 68XG5
+	{0x5152, SWITCHTEC_GEN5, SWITCHTEC_PSX},   //PSX 52XG5
+	{0x5136, SWITCHTEC_GEN5, SWITCHTEC_PSX},   //PSX 36XG5
+	{0x5128, SWITCHTEC_GEN5, SWITCHTEC_PSX},   //PSX 28XG5
+	{0x5200, SWITCHTEC_GEN5, SWITCHTEC_PAX},   //PAX 100XG5
+	{0x5284, SWITCHTEC_GEN5, SWITCHTEC_PAX},   //PAX 84XG5
+	{0x5268, SWITCHTEC_GEN5, SWITCHTEC_PAX},   //PAX 68XG5
+	{0x5252, SWITCHTEC_GEN5, SWITCHTEC_PAX},   //PAX 52XG5
+	{0x5236, SWITCHTEC_GEN5, SWITCHTEC_PAX},   //PAX 36XG5
+	{0x5228, SWITCHTEC_GEN5, SWITCHTEC_PAX},   //PAX 28XG5
+	{0x5300, SWITCHTEC_GEN5, SWITCHTEC_PAXA},  //PAX-A 100XG5
```

Comment:
What does -A here mean? Is it for rev A? If yes I don't think we need it as the rev B chip will use the same device ID.

Reply:
If it's for another variant, I'm good with these patches.

Reply:
Confirmed with PM, PAX-A/PFX-A are indeed different variants for specific markets. Will keep the IDs as-is.